single_tech_samples/databricks/databricks_terraform/Infra/README.md

```diff
@@ -0,0 +1,71 @@
+# Terraform Code for Multi Environment Databricks Medallion Deployment
+
+![Multi Environment Image](../images/architecture.png)
+
+## Overview
+
+**`Infra/modules`** folder has three modules:
+- **`adb-workspace`** - Deploys Databricks workspace
+- **`metastore-and-users`** - Creates Databricks Connector, Creates Storage Account, Give storage access rights to connector, Creates Metastore / Assigns Workspace to Metastore, and Finally Retrieves alls users, groups, and service principals from Azure AD.
+- **`adb-unity-catalog`** - Gives databricks access rights to the connector, Creates containers in the storage account, and creates external locations for the containers. Creates unity catalog and grants permissions user groups. Finally, creates **`bronze` `silver` `gold`** schemas under the catalog and gives the required permissions to the user groups.
+
```

Comment:
If the script fails with an error - lack of permissions or something else - is the script idempotent? Meaning, can we re-run and the script will continue where it left of? Can we make a note about that?

Reply:
There is a note on that in READ.me of Infra, that the end of the doc